content/applications/inventory_and_mrp/barcode/operations/adjustments.rst

```diff
@@ -1,137 +1,168 @@
-=========================================
-Apply inventory adjustments with barcodes
-=========================================
+==============================
+Adjust inventory with barcodes
+==============================
 
-In a warehouse, the recorded inventory counts in the database might not always match the actual,
-real inventory counts. In such cases, inventory adjustments can be made to reconcile the
-differences, and ensure that the recorded counts in the database match the actual counts in the
-warehouse. In Odoo, the *Barcode* app can be used to make these adjustments.
+An *inventory adjustment*, or inventory audit, is the process of verifying the physical stock of
+products against the quantities recorded in the database. Regular audits ensure accurate inventory
+records, prevent stock discrepancies, and maintain efficient operations. In a warehouse setting,
+managers typically assign inventory counts to employees, who then walk to designated locations, scan
+product barcodes, and adjust quantities as needed.
 
-These adjustments can be done in real time using an Odoo-compatible barcode scanner or the Odoo
-mobile app.
+Inventory adjustments can be completed through the **Barcode** application using a compatible
+scanner, or the Odoo mobile app.
 
 .. note::
-   For a list of Odoo-compatible barcode mobile scanners, and other hardware for the *Inventory* and
-   *Barcode* apps, refer to the `Odoo Inventory • Hardware page
+   For a list of Odoo-compatible barcode mobile scanners, and other hardware for the **Inventory**
+   and **Barcode** apps, refer to the `Odoo Inventory • Hardware page
    <https://www.odoo.com/app/inventory-hardware>`_.
 
 .. seealso::
    :doc:`../../inventory/warehouses_storage/inventory_management/count_products`
 
-Enable Barcode app
-==================
+.. tip::
+   Odoo's **Barcode** application provides demo data with barcodes to explore the features of the
+   app. These can be used for testing purposes, and can be printed from the home screen of the app.
+
+   To access this demo data, navigate to the :menuselection:`Barcode app` and click :guilabel:`stock
+   barcodes sheet` and :guilabel:`commands for Inventory` in the information pop-up window above the
+   scanner.
```

Comment:
The UI and image are showing links "demo data sheet" and "barcode" not "stock barcodes sheet" or "commands for Inventory"
```suggestion
   To access this demo data, navigate to the :menuselection:`Barcode app` and click :guilabel:`demo
   data sheet` or :guilabel:`barcodes` in the banner above the scanner.
```